packages/frontend/web/src/app.tsx

```diff
@@ -66,13 +67,37 @@ export function App() {
     languageLoadingPromise = loadLanguage().catch(console.error);
   }
 
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEvent) => {
+      console.log('Key pressed:', e.key);
+      if (
+        e.key === 's' &&
+        (navigator.platform.match('Mac') ? e.metaKey : e.ctrlKey)
+      ) {
+        e.preventDefault();
+        toast.success('Save');
+      }
+    };
+
+    try {
+      document.addEventListener('keydown', handleKeyDown, false);
+    } catch (error) {
+      console.error('Error attaching event listener:', error);
+    }
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyDown, false);
+    };
+  }, []);
```

Comment:
Instead adding a new useEffect hook, you can follow https://github.com/toeverything/AFFiNE/blob/9b620ecbc94e40f49706f22c912d11a4607911da/packages/frontend/core/src/modules/navigation/view/use-register-navigation-commands.ts#L30-L45 and register a hidden command to ignoring incoming `$mod+s` keybindings

Reply:
okay i will fix it.

Reply:
@pengx17 **i am resolve all the issues you are asked for. please kindly check my new pr. thanks**
![2024-04-16 21_19_41-](https://github.com/toeverything/AFFiNE/assets/111266105/0fb1d9a2-9d87-4ab4-9c2d-6eeafac93e89)